homeassistant/components/foscam/camera.py

```diff
@@ -15,7 +30,32 @@
 DEFAULT_NAME = "Foscam Camera"
 DEFAULT_PORT = 88
 
-FOSCAM_COMM_ERROR = -8
+SERVICE_PTZ = "ptz"
+ATTR_MOVEMENT = "movement"
+ATTR_TRAVELTIME = "travel_time"
+
+DEFAULT_TRAVELTIME = 0.125
+
+DIR_UP = "UP"
```

Comment:
Should we make these strings lowercase snakecase? That's our standard for other strings that the user needs to handle.

Reply:
Done.